### ISSUE TITLE
fails to build with brick-1

### ISSUE THREAD

**Issue by juhp** [codedownio/sandwich#64]
sandwich-0.1.0.10 fails to build in current Stackage Nightly:
```
/var/stackage/work/unpack-dir/unpacked/sandwich-0.1.0.10-bfeba0f1a1fe3433068b22c4980eadaf8a4eaea3eeb2c05014f941a805add811/src/Test/Sandwich/Formatters/TerminalUI/AttrMap.hs:151:19: error:
    • No instance for (Data.String.IsString AttrName)
        arising from the literal ‘"logFunction"’
    • In the expression: "logFunction"
      In an equation for ‘logFunctionAttr’:
          logFunctionAttr = "logFunction"
    |
151 | logFunctionAttr = "logFunction"
    |                   ^^^^^^^^^^^^^

/var/stackage/work/unpack-dir/unpacked/sandwich-0.1.0.10-bfeba0f1a1fe3433068b22c4980eadaf8a4eaea3eeb2c05014f941a805add811/src/Test/Sandwich/Formatters/TerminalUI/AttrMap.hs:176:23: error:
    • No instance for (Data.String.IsString AttrName)
        arising from the literal ‘"fieldName"’
    • In the expression: "fieldName"
      In an equation for ‘constructorNameAttr’:
          constructorNameAttr = "fieldName"
    |
176 | constructorNameAttr = "fieldName"
    |                       ^^^^^^^^^^^
```

**Comment by juhp** [comment]
I am guessing this caused by brick-1.1

**Comment by thomasjm** [comment]
Right, this is also tracked on https://github.com/commercialhaskell/stackage/issues/6680.

I'm slightly confused because on the latest nightly I see, `nightly-2022-09-04`, there is `brick-0.73` and Sandwich builds successfully. Is this happening because you're trying to switch Stackage to `brick-1.x` ?

**Comment by juhp** [comment]
Exactly yes - i had missed the Stackage ticket.
This is blocking hledger-1.27 from going into nightly

**Comment by juhp** [comment]
So firstly current sandwich should have an upperbound on brick.
But do you think you will be able to do a release to accommodate brick-1 soon?
I feel it would be good to have brick-1 in the coming LTS 20.

**Comment by thomasjm** [comment]
Are you sure about `hledger-1.27`? I don't think it has a dependency on `sandwich`.

Yeah, I've been meaning to do the `brick-1` migration soon.

Would it help if I released a new point release with the upper bound?

**Comment by juhp** [comment]
hledger-1.27 requires brick-1

**Comment by juhp** [comment]
Well you could just do a Hackage revision for the upperbound but brick-1 would still be blocked by this.

**Comment by thomasjm** [comment]
Just released `sandwich-0.1.0.11` which should support the new `brick`. Want to try it?

**Comment by juhp** [comment]
Great, thanks so much
